src/pages/SitePolicy/policies/CookiePolicy.tsx

```diff
@@ -0,0 +1,148 @@
+    Third party cookies enable third party features or functionality to be provided on or through the website
+    (e.g. like advertising, interactive content and analytics).
+    The parties that set these third party cookies can recognize your computer both when it visits the website
+    in question and also when it visits certain other websites.
+    <br/><br/>
+    <h6>Why do we use cookies?</h6>
+    We use first and third party cookies for several reasons.
+    Some cookies are required for technical reasons in order for our Websites to operate, and
+    we refer to these as "essential" or "strictly necessary" cookies.
+    Other cookies also enable us to track and target the interests of our users to enhance the experience
+    on our Online Properties.
+    Third parties serve cookies through our Websites for advertising, analytics and other purposes.
+    This is described in more detail below.
+    <br/>
+    The specific types of first and third party cookies served through our Websites and
+    the purposes they perform are described below (please note that the specific cookies served may vary depending
+    on the specific Online Properties you visit):
+    <br/><br/>
+    <h6>How can I control cookies?</h6>
+    You have the right to decide whether to accept or reject cookies.
+    You can exercise your cookie rights by setting your preferences in the Cookie Consent Manager.
+    The Cookie Consent Manager allows you to select which categories of cookies you accept or reject.
+    Essential cookies cannot be rejected as they are strictly necessary to provide you with services.
+    <br/>
+    The Cookie Consent Manager can be found in the notification banner and on our website.
+    If you choose to reject cookies, you may still use our website though your access to some functionality
+    and areas of our website may be restricted. You may also set or amend your web browser controls to accept
+    or refuse cookies. As the means by which you can refuse cookies through your web browser controls vary
+    from browser-to-browser, you should visit your browser's help menu for more information.
+    <br/>
+    In addition, most advertising networks offer you a way to opt out of targeted advertising.
+    If you would like to find out more information, please visit http://www.aboutads.info/choices/
+    or http://www.youronlinechoices.com.
+    <br/>
+    The specific types of first and third party cookies served through our Websites and the purposes
+    they perform are described in the table below (please note that the specific cookies served may
+    vary depending on the specific Online Properties you visit):
+    <br/>
+    Essential website cookies:
+    <br/>
+    These cookies are strictly necessary to provide you with services available through our
+    Websites and to use some of its features, such as access to secure areas.
+    <br/>
+    None
+    {/*Name:*/}
```

Comment:
What are these comments for?

Reply:
I do not know what cookies we are using so they are comment out until we get a finalized list of cookies we use for this product.